data/map.txt

```diff
@@ -9681,9 +11083,40 @@ system Eteron
 		distance 1751.6
 		period 2523.74
 	object
-		sprite planet/gas17-b
-		distance 2229.85
-		period 3624.98
+		sprite planet/gas17-b
+		distance 2229.85
+		period 3624.98
+
+system Faerilloumier-iking-kicklissnika-puol
+	pos 36.361067 154.935
+	government Xapleaux
+	link Dsik-geromil-jek
+	asteroids "small rock" 16 3.3264
+	asteroids "medium rock" 9 6.3504
+	asteroids "large rock" 54 4.7088
+	asteroids "small metal" 5 4.7952
+	asteroids "medium metal" 23 2.5488
+	asteroids "large metal" 6 5.4
+	minables gold 13 8.16253
+	minables iron 23 4.48663
+	minables titanium 25 6.84244
+	haze _menu/haze-cloudbright
+	habitable 115184.45
+	belt 1427
+	object
+		sprite star/black-hole
```

Comment:
This should probably have the same effects that Sagittarius A* has, no? it was surprising that this black hole didn't suck me in and crush me into nothing

Reply:
Sagittarus is the biggest BH in the galaxy, this is a random one so if it had an effect it should be way smaller

Reply:
A stellar black hole is still a stellar black hole. As far as our ships are concerned, I'd say "only a few stellar masses" is about as deadly as "several thousand stellar masses." The lore is that our gravitational nullification systems that allow ships to launch from planets also provides strong protection from black holes. As in with it, they are "only" deadly over a short duration of exposure, as opposed to deadly instantaneously.

Reply:
I actually want to make this one as dangerous as Sag A is now, and then bump Sag A up to be *much* more dangerous.

Reply:
> I actually want to make this one as dangerous as Sag A is now, and then bump Sag A up to be _much_ more dangerous.

once you fall into the gravity well of Sag A you're dead whatever you do pretty much, so how much dmg is done per second doesn't matter (it's less painful if more damage because it takes less long to die I think), but I the gravity pull could indeed be stronger

Reply:
You absolutely can escape from Sag A* if you have a high enough thrust-mass ratio, it is not deadly unless you spend a prolonged amount of time sitting on top of it.